tests/Integration/Behaviour/Features/Scenario/Order/order_from_bo.feature

```diff
@@ -263,7 +263,6 @@ Feature: Order from Back Office (BO)
       | name          | Mug Today is a good day |
       | amount        | 2                       |
       | price         | 16                      |
-      | free_shipping | true                    |
```

Comment:
Why remove this line ?

Reply:
There has been a change in the function and this key is not mandatory anymore

Reply:
:ok_hand: